setup.py

```diff
@@ -15,7 +20,6 @@ def get_version(file='./pls/static/version'):
     name="plsbuild",
     version=get_version(),
     packages=find_packages(),
-    install_requires=get_reqs(),
```

Comment:
Style nit: I would still call this function (i.e. keep this line), but, in the function, just confirm there's no `requirements.txt`.

So that once it is added our package build fails with a clear error message!